var/spack/repos/builtin/packages/neurodamus-model/package.py

```diff
@@ -150,8 +183,45 @@ def setup_environment(self, spack_env, run_env):
             run_env.prepend_path('PYTHONPATH', self.prefix.python)
             run_env.set('NEURODAMUS_PYTHON', self.prefix.python)
 
+
 @contextmanager
 def profiling_wrapper_on():
     os.environ["USE_PROFILER_WRAPPER"] = "1"
     yield
     del os.environ["USE_PROFILER_WRAPPER"]
+
+
+# Aux funcs
+# ---------
+def copy_all(src, dst, copyfunc=shutil.copy):
+    """Copies/processes all files in a src dir against a destination dir"""
+    isdir = os.path.isdir
+    for name in os.listdir(src):
+        pth = join_path(src, name)
+        isdir(pth) or copyfunc(pth, dst)
+
+
+def symlink2(src, dst):
+    """Simple alternative to symlink, copy compat"""
+    if os.path.isdir(dst):
+        dst_dir = dst
+        dst = join_path(dst, os.path.basename(src))
+    else:
+        dst_dir = os.path.dirname(dst)
+    src = os.path.relpath(src, dst_dir)  # update path relation
+    # Silently replace links, just like copy replaces files
+    if os.path.islink(dst):
+        os.remove(dst)
+    os.symlink(src, dst)
+
+
+def filter_out(src, dst):
+    """Remove src from dst, copy compat"""
+    fname = join_path(dst, os.path.basename(src))
+    if os.path.exists(fname):
+        os.remove(join_path(dst, os.path.basename(src)))
```

Comment:
something went wrong where
```suggestion
        os.remove(fname)
```